docs/docs/tutorials/orca_k8s/orca_k8s_tutorial.md

```diff
@@ -0,0 +1,563 @@
+* `remote_dir`: directory on NFS for the dataset (see __Section 4__) and saving the model.
+
+
+### 6.1.2 K8s-Cluster
+
+Before running the example on `k8s-cluster` mode, you should:
+* On the __Client Container__:
+    1. Please call `init_orca_context` at very begining part of each Orca program.
+        ```python
+        from bigdl.orca import init_orca_context, stop_orca_context
+
+        conf={
+              "spark.kubernetes.driver.volumes.persistentVolumeClaim.nfsvolumeclaim.options.claimName":"nfsvolumeclaim",
+              "spark.kubernetes.driver.volumes.persistentVolumeClaim.nfsvolumeclaim.mount.path": "/bigdl/nfsdata",
+              "spark.kubernetes.executor.volumes.persistentVolumeClaim.nfsvolumeclaim.options.claimName":"nfsvolumeclaim",
+              "spark.kubernetes.executor.volumes.persistentVolumeClaim.nfsvolumeclaim.mount.path": "/bigdl/nfsdata",
+              "spark.kubernetes.authenticate.driver.serviceAccountName":"spark",
+              "spark.kubernetes.file.upload.path":"/bigdl/nfsdata/"
+              }
+
+        init_orca_context(cluster_mode="k8s-cluster", num_nodes=2, cores=2, memory="2g",
+                          master="k8s://https://<k8s-apiserver-host>:<k8s-apiserver-port>", 
+                          container_image="intelanalytics/bigdl-k8s:latest",
+                          penv_archive="file:///bigdl2.0/data/sgwhat/environment.tar.gz",
+                          extra_python_lib="/bigdl/nfsdata/model.py", conf=conf)
+        ```
+
+        When running Orca programs on `k8s-cluster` mode, please use the following additional configuration propeties: 
+        * `spark.kubernetes.driver.volumes.persistentVolumeClaim.nfsvolumeclaim.options.claimName`: specify the claim name of `persistentVolumeClaim` with volumnName `nfsvolumeclaim` to mount `persistentVolume` into driver pod;
+        * `spark.kubernetes.driver.volumes.persistentVolumeClaim.nfsvolumeclaim.mount.path`: add volumeName `nfsvolumeclaim` of the volumeType `persistentVolumeClaim` to driver pod on the NFS path specified in value;
+        * `spark.kubernetes.executor.volumes.persistentVolumeClaim.nfsvolumeclaim.options.claimName`: specify the claim name of `persistentVolumeClaim` with volumnName `nfsvolumeclaim` to mount `persistentVolume` into executor pods;
+        * `spark.kubernetes.executor.volumes.persistentVolumeClaim.nfsvolumeclaim.mount.path`: add volumeName `nfsvolumeclaim` of the volumeType `persistentVolumeClaim` to executor pods on the NFS path specified in value;
+        * `spark.kubernetes.authenticate.driver.serviceAccountName`: the service account for driver pod;
+        * `spark.kubernetes.file.upload.path`: the path to store files at spark submit side in cluster mode;
+    
+    2. Using Conda to install BigDL and needed Python dependency libraries (see __Section 3__).
+
+    3. Pack the current activate Conda environment to an archive.
+        ```
+        conda pack -o environment.tar.gz
```

Comment:
better put these two steps before the first one; otherwise environment.tar.gz is awkward in step 1